docs/content/devguide/075-programmatic-api.adoc

```diff
@@ -162,8 +165,17 @@ VectorStatisticsQueryBuilder<Object> statisticsQueryBuilder = VectorStatisticsQu
 // Create the query by vector statistics type factory
 QueryByVectorStatisticsTypeFactory queryByStatTypeFactory = statisticsQueryBuilder.factory();
 
+// Create the bounding box query builder
+FieldStatisticsQueryBuilder<Envelope> builder = queryByStatTypeFactory.bbox();
+
+// Specify the geometry field name
+builder.fieldName("the_geom");
+
+// Specify the type name
+builder.dataType(pointTypeAdapter.getTypeName());
```

Comment:
keep in mind, field name and data type are optional, they're basically filters and it will aggregate the bounding box of anything matching the query - so while data type seems likely to be desired, maybe field name isn't if there are mutliple geometry fields in the type and you want the bounding box of everything.  Anyways, just perhaps worth a comment in the example to be clear how it behaves?